### PR TITLE
Update protobuf codec doc

### LINKED ISSUE #772: Update CUSTOM_PROTOBUF_CODECS.md for v0.25
The code in this document will be out of date after the changes in v0.25.

### PR DESCRIPTION
**Pull request by willclarktech**
Closes #772

### CODE REVIEW

File: packages/stargate/CUSTOM_PROTOBUF_CODECS.md [hunk 1]
```diff
@@ -2,29 +2,33 @@
 
 As of [v0.40](https://github.com/cosmos/cosmos-sdk/releases/tag/v0.40.0), the
 Cosmos SDK uses
-[protocol buffers](https://developers.google.com/protocol-buffers) as its
-standard serialization format for blockchain state and wire communication.
-CosmJS by default supports protocol buffer serialization for many of the
-standard queries and messages defined by the Cosmos SDK, as well as
+[protocol buffers](https://developers.google.com/protocol-buffers) (also known
+as "protobufs") as its standard serialization format for blockchain state and
```
Comment (webmaster128) [inline packages/stargate/CUSTOM_PROTOBUF_CODECS.md:6]: isn't it

```suggestion
as "protobuf") as its standard serialization format for blockchain state and
```

**Review by webmaster128 — COMMENTED** [review]
Very nice

File: packages/stargate/CUSTOM_PROTOBUF_CODECS.md (hunk 1 above)
Comment (willclarktech) [reply packages/stargate/CUSTOM_PROTOBUF_CODECS.md:6]: Hm, the grammar is weird but the opening line of [the wikipedia entry](https://en.wikipedia.org/wiki/Protocol_Buffers) seems to think "Protocol Buffers" is a singular proper noun equivalent to "Protobuf". But then uses "are" later on. :S

Comment (willclarktech) [reply packages/stargate/CUSTOM_PROTOBUF_CODECS.md:6]: GitHub agrees with you: https://github.com/protocolbuffers/protobuf#overview

**Review by webmaster128 — APPROVED** [review]

**Pull request merged**